wasmbinding/test/custom_message_test.go

```diff
@@ -723,6 +728,103 @@ func (suite *CustomMessengerTestSuite) TestAddRemoveSchedule() {
 	suite.NoError(err)
 }
 
+func (suite *CustomMessengerTestSuite) TestBurnTokens() {
+	// add NTRN to the contract
+	senderAddress := suite.ChainA.SenderAccounts[0].SenderAccount.GetAddress()
+	coinsAmnt := sdk.NewCoins(sdk.NewCoin(params.DefaultDenom, math.NewInt(int64(10_000_000))))
+	bankKeeper := suite.neutron.BankKeeper
+	err := bankKeeper.SendCoins(suite.ctx, senderAddress, suite.contractAddress, coinsAmnt)
+	suite.NoError(err)
+
+	suite.ConfigureTransferChannel()
+
+	// add IBC denom to the contract
+	timeoutHeight := clienttypes.NewHeight(1, 110)
+
+	// Create Transfer Msg
+	transferMsg := transfertypes.NewMsgTransfer(suite.TransferPath.EndpointB.ChannelConfig.PortID,
+		suite.TransferPath.EndpointB.ChannelID,
+		sdk.NewCoin(params.DefaultDenom, math.NewInt(100)),
+		suite.ChainB.SenderAccounts[0].SenderAccount.GetAddress().String(),
+		strings.TrimSpace(suite.contractAddress.String()),
+		timeoutHeight,
+		0,
+		"",
+	)
+
+	// Send message from chainB to chainA
+	res, err := suite.TransferPath.EndpointB.Chain.SendMsgs(transferMsg)
+	suite.Require().NoError(err)
+
+	// Relay transfer msg to Neutron chain
+	packet, err := ibctesting.ParsePacketFromEvents(res.GetEvents())
+	suite.Require().NoError(err)
+
+	//nolint:errcheck // this will return an error for multi-hop routes; that's expected
+	suite.Require().NoError(suite.TransferPath.RelayPacket(packet))
```

Comment:
> this will return an error

> suite.Require().NoError

huh?

Reply:
It's a copupasted comment from a different testtuit, removed